### PR TITLE
Kruskal Algorithm implementation in Dart

### LINKED ISSUE #2200: Kruskal Algorithm
- [ ] C - @coderprasukj
- [x] C++
- [ ] C# - @chandrikadeb7
- [ ] CoffeeScript
- [ ] Dart - @Swarnimashukla
- [ ] Go
- [x] Java
- [ ] JS - @shreyakapoor08
- [ ] Kotlin - @garimasingh128
- [ ] Php - @prince-09
- [x] Python - @Manish-cloud
- [x] Readme
- [ ] Ruby 
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by Swarnimashukla**
Fixes #2200 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Kruskal Algorithm implementation in Dart

#### Languages Used:
- Dart

#### Files Added:
- Kruskal_Algorithm.dart

@jainaman224  @asha15 @raghav-dalmia @SakshiShreya @shreyakapoor08

### CODE REVIEW

**Comment by Swarnimashukla** [comment]
@jainaman224  @asha15  please review this PR

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+(1) form a tree that includes every vertex
+(2) has the minimum sum of weights among all the trees that can be
+    formed from the graph
+ */
+
+import 'dart:io';
+
+var INT_MAX = 9223372036854775807;
+
+int find( i, parent )
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:14]: Remove space after ( and before )

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+{
+    while ( parent[i] != i ) {
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:16]: Remove space after ( and before )

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        i = parent[i];
+    }
+    return i;
+}
+
+void union1( i, j, parent )
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:22]: Remove space after ( and before )

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+{
+    var a = find( i, parent );
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:24]: Remove space after ( and before ) . Do this throughout the file

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  \        |        /
+   \       |       /
+    \      2      /
+     6     |     8
+      \    |    /
+       \   |   /
+        \  |  /
+         \ | /
+          (4)
+
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:142]: Remove this line

File: Kruskal_Algorithm/Kruskal_Algorithm.dart [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+    ];
+Output:
+
+ Edge: 0-1 cost: 3
+ Edge: 1-2 cost: 1
+ Edge: 1-5 cost: 4
+ Edge: 5-4 cost: 2
+ Edge: 5-3 cost: 5
+ Minimum Weight is 15
+
```
Comment (somya-kapoor) [inline Kruskal_Algorithm/Kruskal_Algorithm.dart:160]: Remove this line

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by Swarnimashukla** [comment]
Yes I will submit my final code by  Tomorrow morning

**Comment by Swarnimashukla** [comment]
@somya-kapoor I have made recommended changes please go through it .

**Pull request merged**